src/resources/views/crud/fields/inc/translatable_icon.blade.php

```diff
@@ -1,3 +1,19 @@
-@if (method_exists($crud->model, 'isTranslatableAttribute') && ($crud->model->isTranslatableAttribute($field['name']) || (isset($field['store_in']) ? $crud->model->isTranslatableAttribute($field['store_in']) : false)) && config('backpack.crud.show_translatable_field_icon'))
-<i class="la la-flag-checkered pull-{{ config('backpack.crud.translatable_field_icon_position') }}" style="margin-top: 3px;" title="This field is translatable."></i>
-@endif
\ No newline at end of file
+@php
+    // if field name is array we check if any of the arrayed fields is translatable
+    $translatable = false;
+    if($crud->model->translationEnabled()) {
+        foreach((array) $field['name'] as $field_name){
+            if($crud->model->isTranslatableAttribute($field_name)) {
+                $translatable = true;
+            }
+        }
+
+        if(isset($field['store_in']) && $crud->model->isTranslatableAttribute($field['store_in'])) {
+                $translatable = true;
+        }
```

Comment:
Hmm I'm not sure I get this. Could you please explain why this is here, what use case it covers? Is it when it's a fake field... and the `extras` column is translatable? If so, I'm ok with it but let's add a comment to explain that for future us.

Reply:
But I do have a question. Won't that mean that ALL fake fields become translatable? And in this case... I'm not sure that's the best thing to do. What if you want to have a fake but non-translatable field?

PS. Yes I do know that this `store_in` thing was here before this PR 😂   I'm aware of the hypocrisy, I'm just questioning what decisions "past us" have made 😂

Reply:
That being said, since it looks like this PR doesn't change the behaviour for the worst (please confirm), I believe it can be merged as-is. But there's probably a discussion we need to have about this... non-translatable fake fields... is that even possible now? _Should it_ be possible? Is it _possible_ to _make it possible_? 😂

Reply:
@tabacitu I think this is correct, the validation is `isset($field['store_in']) && $crud->model->isTranslatableAttribute($field['store_in'])` so, if the field is fake, and that fake field is translatable, it shows a flag on the field.

For instance;
- `extras` helds a `title`
- `extras_translatable` helds a `description` and a `content`

For;
- `title`
`store_in = extras`, `extras` is NOT translatable, so, no flag
- `description` or `content`
`store_in = extras_translatable`, `extras_translatable` IS translatable, so flag 🏁

Reply:
Right. Right right.

Also, I think there's one thing missing here that I didn't take into consideration - the translatable values fall back to the default.

So in your example above, if you only used one column (`extras_translatable`) for both translatable and non-translatable fake fields, you could use a field there that _technically_ doesn't need to be translatable, because if it's empty it will be populated with the value in the main language.

So it looks like we're all good here, sorry for sidetracking this issue. Merging!